runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -74,7 +74,7 @@ struct TR_JitPrivateConfig;
 struct TR_MethodToBeCompiled;
 template <typename T> class TR_PersistentArray;
 typedef J9JITExceptionTable TR_MethodMetaData;
-class ClientSessionHT; // JITaaS TODO: maybe move all JITServer specific stuff in an extension for CompInfo
+class ClientSessionHT; // JITServer TODO: maybe move all JITServer specific stuff in an extension for CompInfo
 
 #if defined(JITSERVER_SUPPORT)
```

Comment:
This PR is used to only to update `JITServerHelpers` related functions to be ported to the `master` branch.  This forward declaration is not required. The difference shown here is because this PR contains the change from #6865 which replaces all 'JITaaS' with 'JITServer'. Once #6865 is merged and this PR is rebased. The difference will not show up.